### PR TITLE
Install last patch

### PR DESCRIPTION
**Pull request by ebattat**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description

Installing only the last non approved patch to avoid installing incorrect patch as following:
```
$ oc get InstallPlan -n openshift-storage
NAME            CSV                          APPROVAL   APPROVED
install-1   mcg-operator.v4.12.9-1  Manual          false
install-2  ocs-operator.v4.13.4-2   Manual          false
```

### CODE REVIEW

File: benchmark_runner/common/ocp_resources/create_ocp_resource_operations.py [hunk 1]
```diff
@@ -83,46 +83,46 @@ def wait_for_ocp_resource_create(self, resource: str, verify_cmd: str, status: s
...
+            install_plan = self.__oc.run(cmd=install_plan)
+            install_plan_cmd = """ oc patch InstallPlan -n namespace install_plan -p '{"spec":{"approved":true}}' --type merge""".replace('install_plan', str(install_plan))
+            run_install_plan_cmd = install_plan_cmd.replace('namespace', namespace)
+            self.__oc.run(cmd=run_install_plan_cmd)
+            # verify current status
+            check_status = 'oc get InstallPlan -n namespace -ojsonpath={..spec.approved}'.replace('namespace', namespace).strip()
+            result = self.__oc.run(cmd=check_status).split()[last_index]
+            if result == 'true':
+                if resource == 'odf':
+                    for ind in range(3):
```
Comment (RobertKrawitz) [inline benchmark_runner/common/ocp_resources/create_ocp_resource_operations.py:108]: Why `range(3)`?

Shouldn't you try to match the install plan to the OCP version?

Comment (ebattat) [reply benchmark_runner/common/ocp_resources/create_ocp_resource_operations.py:108]: because in ODF it return 3 options

Comment (RobertKrawitz) [reply benchmark_runner/common/ocp_resources/create_ocp_resource_operations.py:108]: Is it guaranteed to return exactly three options?

**Comment by ebattat** [comment]
The issue is with Github actions

**Pull request closed without merging**